src/DynamoCoreWpf/Views/PackageManager/PackagePathView.xaml.cs

```diff
@@ -29,14 +29,9 @@ public PackagePathView()
             DataContextChanged += PackagePathView_DataContextChanged;
         }
 
-        internal PackagePathView(PackagePathViewModel viewModel)
```

Comment:
you don't need this constructor anymore?

Reply:
I don't think so, I don't see it used anywhere.